spec/factories/user.rb

```diff
@@ -15,6 +15,10 @@
     factory :admin do
       is_admin true
     end
+
+    factory :non_admin do
+      is_admin false
```

Comment:
Yes, I understand that. :)

That is said: the lighter your factory is, the happier you are.

Plus, `build(:user)` says what it does: it builds an user, a non-admin one, without surprising me.

Finally, with your implementation: what would `build(:user)` builds?